xclim/sdba/adjustment.py

```diff
@@ -254,17 +309,35 @@ class QuantileDeltaMapping(EmpiricalQuantileMapping):
     where :math:`F` is the cumulative distribution function (CDF). This equation is valid for multiplicative adjustment.
     The algorithm is based on the "QDM" method of [Cannon2015]_.
 
+    Parameters
+    ----------
+    At init:
+
+    nquantiles : int
+      The number of quantiles to use. Two endpoints at 1e-6 and 1 - 1e-6 will be added.
+    kind : {'+', '*'}
+      The adjustment kind, either additive or multiplicative.
+    group : Union[str, Grouper]
+      The grouping information. See :py:class:`xclim.sdba.base.Grouper` for details.
+
+    In adjust:
+
+    interp : {'nearest', 'linear', 'cubic'}
+      The interpolation method to use when interpolating the adjustment factors. Defaults to "nearest".
+    extrapolation : {'constant', 'nan'}
+      The type of extrapolation to use. See :py:func:`xclim.sdba.utils.extrapolate_qm` for details. Defaults to "constant".
```

Comment:
Maybe in a later PR think about a mechanism to have this docstring in `adjust` itself. Fine as it is at the moment.

Reply:
I thought of it, we could simply patch the class doc to `adjust.__doc__` or in a more fancy way.